### ISSUE TITLE
Pathway gene conflation in synonymizer

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#1474]
Mark Williams has asked me about https://reactome.org/content/detail/R-HSA-909733 and how it's represented in our system. Curiously, if you use the synonymizer to look for `R-HSA-909733` you get a gene: https://arax.ncats.io/?term=R-HSA-909733 , but if you search for `REACT:R-HSA-909733` you get the pathway: https://arax.ncats.io/?term=REACT:R-HSA-909733

**Comment by ericawood** [comment]
What version of KG2 is the synonymizer using for that?

**Comment by ericawood** [comment]
I think there is a problem in the ensembl ETL:
![image](https://user-images.githubusercontent.com/36611732/118894641-7ff2a100-b8b9-11eb-9ce7-e3600ed52a36.png)

**Comment by edeutsch** [comment]
I agree, that looks like the problem. But it is unclear to me how to fix it.

Strange that one ENSG should be stated to have 10 different REACT synonyms.

**Comment by saramsey** [comment]
This looks to be a bug in `ensembl_json_to_kg_json.py`. At line 82 of that script

https://github.com/RTXteam/RTX/blob/8c7509f99bf12a2fdf2512cd5a32a9b6382033f8/code/kg2/ensembl_json_to_kg_json.py#L82

the code appears to assume that every cross-reference in Ensembl for an Ensembl gene, is, in fact, a gene. But clearly if the cross-reference identifier starts with `R-HSA`, it is not a gene, and should instead be handled by a `biolink:participates_in` edge between the Ensembl gene and the relevant Reactome pathway.
